cowamm/profitability/10k_growth/uni_v2_4047194.sql

```diff
@@ -17,15 +17,43 @@ with date_range as (
 ),
 
 -- Finds the uniswap v2 pool address given tokens specified in query parameters (regardless of order)
-pool as (
+pools as (
```

Comment:
Is it possible that a pool is created, then closed, and then created again? If so, i guess this table would pick on all of these created pools. Not sure if it can ever happen but in that case i guess we should pick the latest creation log.